packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1070,6 +1070,12 @@ oie.custom_app.push.sent = Push notification sent
 oie.custom_app.push.resend = Resend push notification
 # {0} refers to the display name of the authenticator
 oie.custom_app.push.warning = Haven't received a push notification yet? Try opening {0} on your phone.
+oie.authenticator.custom_app.method.push.verify.enable.biometrics = Your response was received, but your organization requires biometrics—like a fingerprint or facial scan—for access. Make sure your device meets the following requirements, then try again:
```

Comment:
do know if we need to make any changes to the pseudo-loc file `login_ok_PL.properties` or does that automatically get down somehow? In the PR i was using as a reference there were changes to this english property file and to the pseudoloc lang property file https://github.com/okta/okta-signin-widget/pull/1961/files#diff-9d467ad24f316e447c7d2ac4da53ddb2b1becd3ada0b0fc6447ce9c882cd10f4R719-R724